promql/fuzz.go

```diff
@@ -50,11 +54,19 @@ const (
 // Note that his is not the parser for the text-based exposition-format; that
 // lives in github.com/prometheus/client_golang/text.
 func FuzzParseMetric(in []byte) int {
-	p := textparse.New(in)
-	for p.Next() {
+	p := textparse.New(in, "")
```

Comment:
Would be good to also test the OpenMetrics parser in addition to the current Prometheus metrics parser. Maybe just do the same fuzzing a second time with a `application/openmetrics-text` content type parameter?

Reply:
done